examples/README.md

```diff
@@ -0,0 +1,61 @@
+| CSL_RECEIVER | OT_CSL_RECEIVER | 0 | Enables CSL receiver feature for Thread 1.2. |  |
+| DEBUG | not implemented | 0 | Allows building debug instance. Code optimization is disabled. |  |
+| DHCP6_CLIENT | OT_DHCP6_CLIENT | 0 | Enables support for the DHCP6 client. | The device is able to act as typical DHCP client. Enable this switch on a device that is supposed to request networking parameters from the DHCP server. |
+| DHCP6_SERVER | OT_DHCP6_SERVER | 0 | Enables support for the DHCP6 server. | The device is able to act as typical DHCP server. Enable this switch on a device that is supposed to provide networking parameters to devices with DHCP_CLIENT switch enabled. |
+| DIAGNOSTIC | OT_DIAGNOSTIC | 0 | Enables diagnostic support. | Enable this switch on a device that is tested in the factory production stage. |
+| DISABLE_DOC | not implemented | 0 | Disables building of the documentation. |  |
+| DISABLE_TOOLS | not implemented | 0 | Disables building of tools. |  |
+| DNS_CLIENT | OT_DNS_CLIENT | 0 | Enables support for DNS client. | Enable this switch on a device that sends a DNS query for AAAA (IPv6) record. |
+| DUA | OT_DUA | 0 | Enables the Domain Unicast Address feature for Thread 1.2. |  |
+| DYNAMIC_LOG_LEVEL | not implemented | 0 | Enables the dynamic log level feature. | Enable this switch if OpenThread log level is required to be set at runtime. Read more about logging: https://openthread.io/reference/group/api-logging. |
+| ECDSA | OT_ECDSA | 0 | Enables support for Elliptic Curve Digital Signature Algorithm. | Enable this switch if ECDSA digital signature is used by application. |
+| EXTERNAL_HEAP | OT_EXTERNAL_HEAP | 0 | Enables support for external heap. | Make sure to specify the external heap Calloc and Free functions to be used by the OpenThread stack. Enable this switch if the platform uses its own heap. |
+| IP6_FRAGM | OT_IP6_FRAGM | 0 | Enables support for IPv6 fragmentation. |  |
+| JAM_DETECTION | OT_JAM_DETECTION | 0 | Enables support for Jam Detection. | Enable this switch if a device requires the ability to detect signal jamming on a specific channel. Read more about Jam Detection: https://openthread.io/guides/build/features/jam-detection. |
+| JOINER | OT_JOINER | 0 | Enables support for Joiner. | Enable this switch on a device that has to be commissioned to join the network. Read more about the Joiner API: https://openthread.io/reference/group/api-joiner. |
+| LEGACY | OT_LEGACY | 0 | Enables support for legacy network. |  |
+| LINK_RAW | OT_LINK_RAW | 0 | Enables the Link Raw service. |  |
+| LOG_OUTPUT | not implemented | PLATFORM_DEFINED | Defines if the LOG output is to be created and where it goes. | There are several options available: `NONE`, `DEBUG_UART`, `APP`, `PLATFORM_DEFINED` (default). |
+| MAC_FILTER | OT_MAC_FILTER | 0 | Enables support for the MAC filter. |  |
+| MLE_LONG_ROUTES | OT_MLE_LONG_ROUTES | 0 | Enables the MLE long routes extension (experimental, breaks Thread conformance). |  |
```

Comment:
Is "Thread conformance" the same as Thread Specification?

Reply:
by `breaks Thread conformance` I meant that turning on this feature breaks Thread Specification conformance.

Reply:
Let's make it consistent across switches then, as TIME_SYNC states:

**Note: this feature is incompatible with Thread Specification.**

Which I assume is the same thing.  We should say something like "Enabling this feature breaks conformance to the Thread Specification." for both